roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -27,8 +27,9 @@ debug = {{ nova.logging.debug }}
 # Scheduler #
 scheduler_default_filters = {{ nova.scheduler_default_filters }}
 
-default_availability_zone = {{ nova.default_availability_zone }}
+{% if nova.default_availability_zone is defined -%}
```

Comment:
Would you want to leave something like this in so someone could override from site config, just have the default be not specified?